TophatModules/Sources/TophatServer/TophatServer.swift

```diff
@@ -56,7 +56,15 @@ public final class TophatServer {
 
 		let queryItems = request.queryParams.map { URLQueryItem(name: $0, value: $1) }
```

Comment:
Perhaps we could decode here?

```suggestion
		let queryItems = request.queryParams.map { URLQueryItem(name: $0, value: $1.removingPercentEncoding) }
```

Reply:
Yeap! Tested, that works too